packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -2,45 +2,31 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 
 export type ProgressSlots = {
   /**
-   * The root of the Progress
+   * The root of the Progress. This slot will also function as the Progress's track
    * The root slot receives the `className` and `style` specified directly on the `<Progress>`.
    */
   root: NonNullable<Slot<'div'>>;
-  /**
-   * The title of the Progress.
-   * The label slot receives the styling related to the title associated with the Progress.
-   */
-  label?: Slot<'span'>;
   /**
    * The animated slot of the Progress
    * The bar slot receives the styling related to the loading bar associated with the Progress
    */
   bar?: NonNullable<Slot<'div'>>;
-  /**
-   * The track slot of the Progress
-   * The track slot receives the styling related to the loading bar track associated with the Progress
-   */
-  track?: NonNullable<Slot<'div'>>;
-  /**
-   * The description slot of the Progress
-   * The description slot receives the styling related to the description associated with the Progress
-   */
-  description?: Slot<'span'>;
 };
 
 /**
  * Progress Props
  */
 export type ProgressProps = Omit<ComponentProps<ProgressSlots>, 'size'> & {
   /**
-   * Prop to set whether the Progress is determinate or indeterminate
-   * @default false
+   * Percentage of the operation's completeness of the determinate progress
+   *  in decimal format, numerically between 0 and 1.
    */
-  indeterminate?: boolean;
+  value?: number;
   /**
-   * Percentage of the operation's completeness, numerically between 0 and 100.
+   * Max value of the determinate progress.
```

Comment:
A couple of questions regarding `max`.

1. If we are adding a `max` prop, should we have a `min` prop as well?
2. Should we change the description for value to say "numerically between 0 and `max`" instead of "numerically between 0 and 1".